indexer-app/src/cmr/indexer/data/concepts/subscription.clj

```diff
@@ -25,7 +47,10 @@
           :native-id native-id
           :native-id.lowercase (string/lower-case native-id)
           :user-id user-id
-          :revision-date revision-date}]
+          :permitted-group-ids permitted-group-ids
+          :revision-date revision-date}
+         doc-for-non-deleted
+          (merge {:metadata-format (name (mt/format-key format))} doc-for-deleted)]
     (if deleted
       doc-for-deleted
-      (assoc doc-for-deleted :metadata-format (name (mt/format-key format))))))
+      doc-for-non-deleted)))
```

Comment:
why do we make this change? The new code is not doing anything different from the old code.

Reply:
I had more things to merge to and didn't want this assoc to be too long. but then it turns out not necessary, so either way, is fine.